ee/tables/tablewrapper/tablewrapper.go

```diff
@@ -0,0 +1,67 @@
+package tablewrapper
+
+import (
+	"context"
+	"fmt"
+	"log/slog"
+	"time"
+
+	"github.com/kolide/launcher/ee/gowrapper"
+	"github.com/kolide/launcher/pkg/traces"
+	"github.com/osquery/osquery-go/plugin/table"
+)
+
+const DefaultTableTimeout = 2 * time.Minute
```

Comment:
```suggestion
const DefaultTableTimeout = 4 * time.Minute
```

I suspect some are slow, so let's start higher maybe?